WalletWasabi.Fluent/Extensions/FriendlyExceptionMessageExtensions.cs

```diff
@@ -27,6 +27,7 @@ public static string ToUserFriendlyString(this Exception ex)
 		{
 			HwiException hwiEx => GetFriendlyHwiExceptionMessage(hwiEx),
 			HttpRequestException httpEx => GetFriendlyHttpRequestExceptionMessage(httpEx),
+			UnauthorizedAccessException uEx => $"Wasabi was unable to perform this action due to a lack of permission. {uEx.Message}",
```

Comment:
nit question: 

Are we sure `{uEx.Message}` will never contain any unexpected technical message? The message before it wouldn't be enough?

Reply:
> Are we sure {uEx.Message} will never contain any unexpected technical message?

Unfortunately, no, I'm not sure.
I tried to write a general sentence for the first one. The second is to cover the different scenarios and to help the devs to identify the problem.
But we have a `LogError` for that, so I think we can remove that extra sentence.

WDYT? `LogError` is enough for us, right?

Reply:
A message on the UI side should never be for the help of a developer. Yeah, the log is enough.